client/actions/index.js

```diff
@@ -1,4 +1,23 @@
-import request from 'superagent'
+import axios from 'axios'
```

Comment:
Let's switch this back to `superagent`. I'm not sure who or why it was replaced with `axios`.